contracts/libraries/CreateCall.sol

```diff
@@ -0,0 +1,26 @@
+pragma solidity ^0.5.0;
+
+
+/// @title Create Call - Allows to use the different create opcodes to deploy a contract
+/// @author Richard Meissner - <richard@gnosis.io>
+contract CreateCall {
+    event ContractCreation(address newContract);
+
+    function performCreate2(uint256 value, bytes memory deploymentData, bytes32 salt) public returns(address newContract) {
```

Comment:
I guess `ProxyFactory` will be refactored then, right?

Reply:
No, proxy factory will stay as is, since there is the special logic for the init part and quite some other functions.